erpnext/healthcare/doctype/patient_appointment/patient_appointment.json

```diff
@@ -14,11 +14,15 @@
   "patient_name",
   "patient_sex",
   "patient_age",
-  "inpatient_record",
   "column_break_1",
+  "inpatient_record",
+  "company",
+  "service_unit",
   "status",
-  "procedure_template",
+  "section_break_11",
```

Comment:
Let's also retain the form layout here. Only the company field should be in the first section

**Field Ordering introduced in this PR:**

![layout-pr](https://user-images.githubusercontent.com/24353136/80035795-3eed5900-850e-11ea-9efe-7d1b7382affd.png)

**Existing Field Ordering:**

![layout-now](https://user-images.githubusercontent.com/24353136/80035838-54628300-850e-11ea-8478-b4c28f61db8c.png)